kate-website2.css

```
@@ -0,0 +1,279 @@
+body {margin:0;}
```

It's best practice to format code consistently (see suggestion)- to help with this look into linters like Prettier, but it's good to get into the habit of spotting these things as it improves your eye for detail.

```suggestion
body { 
    margin: 0;
}
```